refresh-data.sh

```diff
@@ -64,4 +64,6 @@ if [ "$(git status legislative executive build_output.txt --porcelain)" ]; then
   git commit -a -m "Rebuild using new Wikidata data"
 fi
 
-#git push -u origin $BRANCH --force-with-lease
+if [ "$BRANCH" ]; then
+  git push -u origin $BRANCH --force-with-lease
```

Comment:
Note that `--force-with-lease` will fail on the first push with "stale info" (either because the branch in the remote or the remote-tracking branch doesn't exist yet - I'm not sure which exactly triggers that, but it doesn't make much difference in this instance). To deal with this there could be two cases here, based on:

`git rev-parse --verify "$BRANCH"@{u} > /dev/null 2>&1`

... one of which does:

`git push -u origin $BRANCH`

... and the other of which does:

`git push --force-with-lease origin $BRANCH`

Reply:
Are you sure?

```
[alex@localhost pe]$ git push -u origin master:silly-branch --force-with-lease 
Total 0 (delta 0), reused 0 (delta 0)
To github.com:everypolitician/proto-commons-peru.git
 * [new branch]      master -> silly-branch
Branch 'master' set up to track remote branch 'silly-branch' from 'origin'.
```

I'd been using `--force-with-lease` when creating new branches previously quite happily. Could it be something changed in some version of git? (I'm on 2.17.1)

Reply:
Yes, it seems this has been changed! I'm on git 2.7.4 - in between my version and yours was: https://github.com/git/git/commit/64ac39af7020f3a8b - that landed in 2.9.4.

Still, I suspect a lot quite a lot of people will still be on pre 2.9.4 versions of git, so I don't think we can rely on the nice new behaviour :disappointed: